src/Submission/ProjectSubmission.jsx

```diff
@@ -81,8 +81,9 @@ class ProjectSubmission extends React.Component {
         !useArboristUI
         || (isRootUrl(project) && userHasSheepdogProgramAdmin(userAuthMapping))
         || (isProgramUrl(project) && userHasSheepdogProjectAdmin(userAuthMapping))
-        || userHasMethodOnProject('create', project, userAuthMapping)
-        || userHasMethodOnProject('update', project, userAuthMapping)
+        || userHasMethodForServiceOnProject('create', '*', project, userAuthMapping)
+        || userHasMethodForServiceOnProject('update', '*', project, userAuthMapping)
```

Comment:
I think this will now require the user to have literal `service: '*'` in the relevant policy role... I'm not sure this is what is intended?

Reply:
right, so my understanding is if a user is able to submit to sheepdog then he/she must have either of the following for a project:
1. `{service: *, method: create/update}`
2. `{service: *, method: *}`
3. `{service: sheepdog, method: *}` (?)
4. `{service: sheepdog, method: create/update}` (??? - do we even have this)

Does this hold? 🤔

Reply:
Yup yup, * is supposed to match anything. So I think if you add the `|| x.service === '*'` in userHasMethodForServiceOnResource (and same for method), then in these checks you can just have 
```
        || userHasMethodForServiceOnProject('create', 'sheepdog', project, userAuthMapping)
        || userHasMethodForServiceOnProject('update', 'sheepdog', project, userAuthMapping)
```

(Not sure what you mean by do we even have this--as in, in the useryamls?)

Reply:
Yeah I know we can have {service: sheepdog, method: create/update}, just not sure if we have this in any user.yaml now